css/pages/index.css

```diff
@@ -203,15 +207,26 @@ div.handles-container {
   border-radius: 0.4em;
 }
 
+#news-feeds-txt {
+  font-size: 1.1em;
+}
```

Comment:
Where are we using this ?? If we are not using then can we please remove it ?

Reply:
we are using this.